ui/raidboss/timeline.js

```diff
@@ -43,11 +43,11 @@ class Timeline {
       return text;
 
     let orig = text;
-    let locale = this.options.Language || 'en';
+    let parserLang = this.options.PraserLanguage || 'en';
     if (replaceKey === 'replaceText')
-      locale = this.options.TimelineLanguage || this.options.Language || 'en';
+      parserLang = this.options.TimelineLanguage || this.options.PraserLanguage || 'en';
     for (let r of this.replacements) {
-      if (r.locale && r.locale != locale)
+      if (r.parserLang && r.parserLang != parserLang)
```

Comment:
This line is a bug that I only caught because I tried this out in game.  This needs to be `if (r.locale && r.locale != parserLang)` (sorry).  This is referring to fields called `locale` inside the replacement block in trigger files.